source/js/buyers-guide/inject-react/index.js

```diff
@@ -1,5 +1,4 @@
 import injectCreepVote from "./creep-vote.js";
-
```

Comment:
might as well restore this newline, should still be prettier-compatible but gives one fewer file changes in the PR